pages/apply-fed.html

```diff
@@ -39,8 +39,10 @@ <h3 class="info-header">What You&rsquo;ll be doing</h3>
 
   <section class="info">
     <h3 class="info-header">Applications</h3>
-    <p class="info-secondary">Applications for our 2018 Frontend Design Apprenticeship will open at the beginning of the year. Be sure to <a href="https://twitter.com/hearsparkbox">follow us on Twitter</a> and check <a href="https://seesparkbox.com/foundry/">The Foundry</a> for updates.</p>
+    <p class="info-secondary">How badly do you want it? What you provide us is very important—this is your opportunity to present your case. Choose your answers wisely, my friend!</p>
```

Comment:
"Choose your answers wisely, my friend." Rubs me the wrong way. Not a big deal. Just feels a bit overly threatening.